.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: false
+
+addons:
+  apt:
+    packages:
+      - oracle-java8-installer
+
+language: java
+
+dist: trusty
+
+jdk:
+  - openjdk8
+
+install: true # skips travis' default installation step which executes gradle assemble.
+
+script: ./gradlew clean build
+
+# The before_cache and the cache steps cache the gradle installation on travis.
+before_cache:
```

Comment:
I will leave it like it is then and just remove the line break? I think it's better to keep build logic at the top since it won't change in the future. To the bottom we can add some more settings etc.